src/js/index.js

```diff
@@ -11,10 +11,11 @@ const sort = {
 	DIFFICULTY_DESCENDING: (a, b) => a.dataset.difficulty < b.dataset.difficulty ? 1 : -1
 };
 
-const contentEl = document.querySelector('#content');
```

Comment:
These edits comes from the PR that updates `xo`, you need to revert these (or run `npm run lint:fix`)

Reply:
Done ✓